action.yml

```diff
@@ -125,19 +125,13 @@ runs:
         labels: |
           org.opencontainers.image.source=https://github.com/${{ inputs.organization }}/${{ inputs.repository }}
 
-    - name: 'Install jq 1.6'
-      uses: dcarbone/install-jq-action@v1.0.1
-      with:
-        version: 1.6
-        force: 'true'
-
-    - uses: edwardgeorge/jq-action@main
-      id: tag
-      with:
-        compact: true
-        raw-output: true
-        input: ${{ steps.meta.outputs.json }}
-        script: '.tags | map(select(test("sha-\\w{8,}"))) | first | match("sha-\\w{8,}") | .string'
+    - name: Pull out SHA tag
+      id: sha
+      shell: bash
+      run: |
+        sha=$(echo '${{ steps.meta.outputs.json }}' | jq -r '.tags | map(select(test("sha-\\w{8,}"))) | first | match("sha-\\w{8,}") | .string')
+        echo "sha=$sha" >> $GITHUB_OUTPUT
```

Comment:
How is `sha` output being used? How was the previous step generating output?

Reply:
I almost removed this, but it is used as an output to this job so that parent workflows can use the build tag (for tag promotion, etc).

The old action used the "output" format and called a `set-output` internally.

Reply:
I forgot to update the output reference (just pushed now), will test one more run.